newm-server/gradlew

```diff
@@ -0,0 +1,249 @@
+#!/bin/sh
```

Comment:
remove

Reply:
Will remove it and add it to .gitignore file